DayrSuryan/DSEthiop6.xml

```diff
@@ -80,14 +88,15 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                            <watermark>No watermarks.</watermark>
                         </support>
                         <extent>
-                           <measure unit="leaf">120</measure>
+                           <measure unit="leaf">I + 120</measure>
                            <dimensions type="outer" unit="mm">
                               <height>182</height>
                               <width>127</width>
                               <depth>25</depth>
                            </dimensions>
 
                         </extent>
+                        <foliation resp="DN DR">The manuscript is foliated in pencil in the centre of the recto.</foliation>
```

Comment:
Foliation in pencil, in the center of the bottom margin. (If you say "foliation", it is clear).